README.md

```diff
@@ -78,10 +78,10 @@ Quick Guide
 ```javascript
 var Queue = require('bull');
 
-var videoQueue = Queue('video transcoding', 6379, '127.0.0.1');
-var audioQueue = Queue('audio transcoding', 6379, '127.0.0.1');
-var imageQueue = Queue('image transcoding', 6379, '127.0.0.1');
-var pdfQueue = Queue('pdf transcoding', 6379, '127.0.0.1');
+var videoQueue = Queue('video transcoding', {redis: {port: 6379, host: '127.0.0.1'}});
+var audioQueue = Queue('audio transcoding');
+var imageQueue = Queue('image transcoding');
+var pdfQueue = Queue('pdf transcoding');
```

Comment:
I think it's clearer to use use `new` in the examples, e.g. `new Queue('pdf transcoding');`, so it's clearer that an object instance is returned.